tasks/main.yml

```diff
@@ -20,6 +20,12 @@
   become: True
   with_items: "{{ beegfs_kmod_preload }}"
 
+- name: Generate /etc/hosts file
```

Comment:
I think this is out of scope for BeeGFS and ought to be done elsewhere, for example in p3-appliances cluster_setup.